systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```
@@ -142,6 +146,52 @@ void testAutoCreationOfCruiseControlTopicsWithResources(ExtensionContext extensi
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String cpu = "2";
+        String nwThroughputInbound = "10000";
+        String nwThroughputOutbound = "20000";
+
+        resourceManager.createResourceWithWait(extensionContext, KafkaTemplates.kafkaWithCruiseControl(clusterName, 3, 3)
+                .editMetadata()
+                .withNamespace(clusterOperator.getDeploymentNamespace())
+                .endMetadata()
+                .editOrNewSpec()
+                    .editCruiseControl()
+                        .withNewBrokerCapacity()
+                            .withCpu(cpu)
+                            .withInboundNetwork(nwThroughputInbound + "KiB/s")
+                            .withOutboundNetwork(nwThroughputOutbound + "KiB/s")
+                        .endBrokerCapacity()
+                    .endCruiseControl()
+                .endSpec()
+                .build());
+
+
+        ConfigMap cm = kubeClient().listConfigMaps(CruiseControlResources.brokerCapacityConfigMapName(clusterName)).get(0);
+
+
+        Map<String, String> keyAndValue = cm.getData();
+        String capacityConfigAsString = keyAndValue.get("capacity.json");
+
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            JsonNode capacityConfigAsJsonNode = objectMapper.readTree(capacityConfigAsString);
+            ArrayNode brokerCapacities = (ArrayNode) capacityConfigAsJsonNode.get("brokerCapacities");
+
+            for (JsonNode brokerCapacityEntry : brokerCapacities) {
+                JsonNode capacity = brokerCapacityEntry.get("capacity");
+                assertThat(cpu, is(capacity.get("CPU").get("num.cores")));
+                assertThat(nwThroughputInbound, is(capacity.get("NW_IN")));
+                assertThat(nwThroughputOutbound, is(capacity.get("NW_OUT")));
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
```

Should this fail the test instead of printing the stack trace?